index.js

```diff
@@ -0,0 +1,53 @@
+let bookCollection = localStorage.getItem('bookCollection')
```

Comment:
- [ ] Use const here instead of let, since the bookCollection is never being reassigned.